src/applications/avs/loaders/avsLoader.js

```diff
@@ -0,0 +1,21 @@
+import { defer } from 'react-router-dom-v5-compat';
+import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/api';
+import { environment } from '@department-of-veterans-affairs/platform-utilities/exports';
+
+const apiBasePath = `${environment.API_URL}/avs/v0`;
+
+export async function avsLoader({ params }) {
```

Comment:
Have you thought about writing tests for loader?

Reply:
I didn't and should have! Added, thanks 🙂